mappings/net/minecraft/world/entity/EntityIdMap.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_5577 net/minecraft/world/entity/EntityIdMap
```

Comment:
This works with more than just IDs, I think this name may be too specific?

Reply:
it deals in IDs and UUIDs, which I argue are a type of IDs. no idea what to name this class and its implementor though, it's all too generic

Reply:
The alternative names proposed in the former PR were `EntityStore`/`SectionEntityStore`, which are not ideal, but I think at least a bit less confusing. I suppose the issue with the name `EntityIdMap` is that it implies that it works solely with IDs- it's not clear what it is mapping those IDs to.